packages/notifi-frontend-client/lib/configuration/NotifiFrontendConfiguration.ts

```diff
@@ -7,6 +7,7 @@ export type NotifiEnvironment =
 export type NotifiEnvironmentConfiguration = Readonly<{
   env: NotifiEnvironment;
   tenantId: string;
+  storageOption: 'LocalForage' | 'InMemory';
```

Comment:
please make this an object, in case we want to add further configuration for storage. Also, let's make it optional

```
storageOption?: Readonly<{
  driverType?: 'LocalForage' | 'InMemory'
}>
```

Reply:
Nice suggestion. Thanks!